test/tests/ordered_and_unordered_lists.text

```diff
@@ -129,3 +129,17 @@ This was an error in Markdown 1.0.1:
 	*	sub
 
 	that
+
+Ordered lists start from initial number:
+
+3. Three
+1. Four
```

Comment:
This should be `4. Four`, no?

Reply:
Only the first number in a Markdown list actually matters. The rest are all based off that, so you can write whatever you want to have it automatically number everything else.